developer-docs-site/docs/tutorials/first-transaction.md

```diff
@@ -70,9 +110,9 @@ Each Aptos account has a unique account address. The owner of that account holds
   </TabItem>
 </Tabs>
 
-## Step 2) REST interface
+## Step 2: REST interface
 
-Aptos exposes a [REST interface][rest_spec] for interacting with the blockchain. While the data from the REST interface can be read directly, the following snippets of code demonstrate a more ergonomic approach. This next set of code snippets demonstrates how to use the REST interface to retrieve ledger data from the FullNode including account and account resource data. It also demonstrates how to use the REST interface for constructing signed transactions represented by JSON formatting.
+ While the data from the REST interface can be read directly, the following code examples demonstrate a more ergonomic approach. The below code examples demonstrate how to use the REST interface to retrieve ledger data from the FullNode including account and account resource data. The code also demonstrates how to use the REST interface for constructing signed transactions represented by JSON formatting.
```

Comment:
nit: A bit redundant: "the following code examples demonstrate a more ergonomic approach. The below code examples demonstrate"